jet-fuel/db/schema.rb

```diff
@@ -1,12 +1,25 @@
-ActiveRecord::Schema.define(version: 20150327064543) do
+# encoding: UTF-8
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# Note that this schema.rb definition is the authoritative source for your
+# database schema. If you need to create the application database on another
+# system, you should be using db:schema:load, not running all the migrations
+# from scratch. The latter is a flawed and unsustainable approach (the more migrations
+# you'll amass, the slower it'll run and the greater likelihood for issues).
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema.define(version: 20150329101805) do
 
   create_table "urls", force: :cascade do |t|
     t.string   "full_url"
     t.string   "short_url"
     t.string   "title"
-    t.integer  "visit_count"
-    t.datetime "created_at",  null: false
-    t.datetime "updated_at",  null: false
+    t.integer  "visit_count", default: 0
+    t.datetime "created_at",              null: false
+    t.datetime "updated_at",              null: false
```

Comment:
Hound should not make comments on schema since this is generated and out of my control. Leaving Schema as is